src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -594,6 +594,57 @@ public void migrateIssuesToFixVersion(String projectKey, String versionName, Str
         session.migrateIssuesToFixVersion(projectKey, versionName, query);
     }
 
+    public boolean progressIssues(List<JiraIssue> issueList, String workflowActionName, String comment, PrintStream console) throws IOException {
```

Comment:
I think it would be better to re-use/split the current progressMatchingIssues instead of almost duplicating it with another parameters.